services/QuillLMS/app/queries/impact_metrics/schools_containing_certain_teachers_query.rb

```diff
@@ -10,7 +11,7 @@ def initialize(teacher_ids:, options: {})
     end
 
     def run
-      runner.execute(query, {teacher_ids: @teacher_ids})
+      runner.execute(query)
```

Comment:
We originally did this because `teacher_ids` could be so long it broke the API.  I take it that that's no longer the case with the new version?

Reply:
Good question.  I will try test that out.

Reply:
@anathomical Thankfully it seems to [work](https://github.com/empirical-org/Empirical-Core/pull/10788#pullrequestreview-1566033240:~:text=longer%20array%20parameters-,fbc1c52,-Merge%20state)